### PR TITLE
Add a command to get detailed Informations about what a server type does.

### PR DESCRIPTION
**Pull request by vinc3nz**

### CODE REVIEW

**Comment by vinc3nz** [comment]
Yay, It's pull request #300

**Comment by itzJOHv** [comment]
useless imo

File: bot/discord/commands/server/docs.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    if(args[1] == "list") {
+        await message.channel.send(helpEmbed);
+        return;
+    }
+
+    const embed = new Discord.MessageEmbed();
+    switch(args[1].toLowerCase()) {
+       case "aio":
+         embed.setDescription("`All In One`, short `aio` is a unique server type that contains several features that distinguish it from other Development(Bot & Codeserver) server types. This type brings all of the other types in a 1 single type and lets user interact with console. [Source: docs.dbh.wtf](https://docs.dbh.wtf)\nRecommend for beginners, as it is easy to use.\n**Do not use aio to host gameservers. It is not allowed and your server will be deleted.**");
```
Comment (Darker-Ink) [inline bot/discord/commands/server/docs.js:32]: I don't think having 3rd party docs in the bot is a good idea imo

Comment (itzJOHv) [reply bot/discord/commands/server/docs.js:32]: then there's no point in this command as each description is partially takes from the community docs and its not good to not mentioning the source

Comment (Darker-Ink) [reply bot/discord/commands/server/docs.js:32]: Then yea the command is not needed. The staff have voted in staff chat to count it as advertising anyways :thinking:

**Comment by vinc3nz** [comment]
I removed the source for community docs. I wrote this myself, I hope it is right

**Review by omxpro — APPROVED** [review]
Seems pretty legit

**Review by lolollllo — APPROVED** [review]
why not

**Review by Darker-Ink — APPROVED** [review]
:+1:

**Review by DEV-DIBSTER1 — APPROVED** [review]
Looks good, merge ready worthy.

**Comment by vinc3nz** [comment]
I know that it may is not perfect, but I think it will help the users. And the people explaining the 10000th time what aio is.. Hehe

**Comment by itzJOHv** [comment]
i dont think users even gonna notice this command

**Comment by vinc3nz** [comment]
`DBH!server count` is also used. So why not

**Pull request merged**